ts-pkgs/peer-server/tests/e2e/client.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -meuo pipefail
+
+PRIVATE_KEY_PATH=long/path/pk
```

Comment:
Maybe we want to pass this in as an ENV var? And check that it is defined here :thinking: